source/Posets/Delta-V-CompletePoset.lagda

```diff
@@ -0,0 +1,206 @@
+
+\begin{code}
+
+module positive-posets {𝓤  𝓦  𝓥 : Universe} (A : Poset 𝓤 𝓦) where
+ open Joins (rel-syntax A)
+ open δ_𝓥-complete-poset 𝓥 A
+ open Universal fe
+ open PosetReasoning A
+
+ _is-strictly-below_ : (x y : ∣ A ∣ₚ) → {i : is-δ_𝓥-complete} → 𝓤 ⊔ 𝓦 ⊔ (𝓥 ⁺) ̇ 
+ _is-strictly-below_ x y = (x ≤[ A ] y) holds ×
+  ((z : ∣ A ∣ₚ) → (y ≤[ A ] z) holds → (P : Ω 𝓥) →
+  (z is-lub-of ((𝟙 + P holds) , δ x z P)) holds → P holds)
+
+ order : {x y : ∣ A ∣ₚ} → x is-strictly-below y → (x ≤[ A ] y) holds
+ order c = pr₁ c
+
+ sup-condition : {x y : ∣ A ∣ₚ} → x is-strictly-below y →
+  ((z : ∣ A ∣ₚ) → (y ≤[ A ] z) holds → (P : Ω 𝓥) →
+  (z is-lub-of ((𝟙 + P holds) , δ x z P)) holds → P holds)
+ sup-condition c = pr₂ c
+
+ strictly-below-implies-non-trivial : (x y : ∣ A ∣ₚ) → is-δ_𝓥-complete → (x is-strictly-below y)
+  → (x ≤[ A ] y) holds × ¬ (x ＝ y)
+ pr₁ (strictly-below-implies-non-trivial x y i c) = order c
+ pr₂ (strictly-below-implies-non-trivial x y i c) p =
+  𝟘-induction (sup-condition c y (≤-is-reflexive A y) ((𝟘{𝓥} , 𝟘-is-prop {𝓥})) ((g , h)))
+   where
+    g : (y is-an-upper-bound-of ((𝟙 + ((𝟘 , 𝟘-is-prop) holds)) , δ x y (𝟘 , 𝟘-is-prop))) holds
+    g (inl ⋆) = order c
+
+    h : (Ɐ u ꞉ (upper-bound ((𝟙 + ((𝟘 , 𝟘-is-prop) holds)) , δ x y (𝟘 , 𝟘-is-prop))) ,
+     y ≤[ A ] (pr₁ u)) holds
+    h u = y ＝⟨ p ⁻¹ ⟩ₚ pr₂ u (inl ⋆)
+
+\end{code}
+
+We could show that if the converse holds then so does LEM in 𝓥.
+
+\begin{code}
+
+ transitivity-lemma : (x y z : ∣ A ∣ₚ) → is-δ_𝓥-complete →
+  (((x ≤[ A ] y) holds × y is-strictly-below z) → x is-strictly-below z) ×
+  ((x is-strictly-below y × (y ≤[ A ] z) holds) → x is-strictly-below z)
+ pr₁ (transitivity-lemma x y z i) r = (≤-is-transitive A x y z (pr₁ r) (order (pr₂ r)) , h)
+  where
+   h : (w : ∣ A ∣ₚ) → (z ≤[ A ] w) holds → (P : Ω 𝓥) →
+    (w is-lub-of ((𝟙 + (P holds)) , δ x w P)) holds → P holds
+   h w q P l = sup-condition (pr₂ r) w q P ((a , b))
+    where
+     a : (w is-an-upper-bound-of ((𝟙 + (P holds)) , δ y w P)) holds
+     a (inl ⋆) = ≤-is-transitive A y z w (order (pr₂ r)) q
+     a (inr p) = ≤-is-reflexive A w
+
+     b : (Ɐ u ꞉ (upper-bound ((𝟙 + (P holds)) , δ y w P)) , w ≤[ A ] (pr₁ u)) holds 
+     b u = pr₂ l ((pr₁ u , c))
+      where
+       c : (pr₁ u is-an-upper-bound-of ((𝟙 + (P holds)) , δ x w P)) holds
+       c (inl ⋆) = ≤-is-transitive A x y (pr₁ u) (pr₁ r) (pr₂ u (inl ⋆))
+       c (inr p) = pr₂ u (inr p)
+
+ pr₂ (transitivity-lemma x y z i) r =
+  (≤-is-transitive A x y z (order (pr₁ r)) (pr₂ r) , a)
+   where
+    a : (w : ∣ A ∣ₚ) → rel-syntax A z w holds → (P : Ω 𝓥) →
+      (w is-lub-of ((𝟙 + (P holds)) , δ x w P)) holds → P holds
+    a w q P l = sup-condition (pr₁ r) w (≤-is-transitive A y z w (pr₂ r) q) P l
+
+ is-positive : {i : is-δ_𝓥-complete} → 𝓤 ⊔ 𝓦 ⊔ (𝓥 ⁺) ̇ 
+ is-positive = Σ x ꞉ ∣ A ∣ₚ , (Σ y ꞉ ∣ A ∣ₚ , x is-strictly-below y)
+
+\end{code}
+
+Maybe we add syntax for the strictly below relation...
```

Comment:
Yes!

Reply:
I've never actually made syntax before. I do seem some examples to follow in the Join file but would be open to suggestons. Also, since I have been using \leq should I use < for strictly below? This departs from the notation used in your thesis.